src/main/java/org/tron/core/db/Manager.java

```diff
@@ -1324,6 +1343,27 @@ public void processBlock(BlockCapsule block)
     //reset BlockEnergyUsage
     this.dynamicPropertiesStore.saveBlockEnergyUsage(0);
 
+    //
+    Future<Boolean> future = executorService.submit(() -> {
+      try {
+        if (!block.generatedByMyself) {
+          preValidateTransactionSign(block);
+        }
+      } catch (Exception e) {
+        logger.error("preValidateTransactionSign fail! block info: {}", block.toString(),
+            e.getMessage());
+        return false;
+      }
+      return true;
+    });
+    while (!done) {
+      try {
+        Thread.sleep(1);
+      } catch (InterruptedException e) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1916&issues=AWgnKDa4rt4Kct_4Nf2G&open=AWgnKDa4rt4Kct_4Nf2G)

[ISSUE_KEY]: AWgnKDa4rt4Kct_4Nf2G